src/makeRobot.js

```diff
@@ -38,7 +38,81 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      // eslint-disable-next-line max-len
+      return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+
+    goForward(step) {
+      if (step === undefined) {
+        this.coords.y++;
+      }
+
+      if (step >= 1) {
+        this.coords.y += step;
+      }
+
+      return this;
+    },
+
+    goBack(step) {
+      if (step === undefined) {
+        this.coords.y--;
+      }
+
+      if (step >= 1) {
+        this.coords.y -= step;
+      }
+
+      return this;
+    },
```

Comment:
you can make your methods shorter using default value for `step`
```suggestion
    goBack(step = 1) {
      if (step > 0) {
        this.coords.y -= step;
      }

      return this;
    },
```